api/v2/guides/publisher-client-side.md

```diff
@@ -1,123 +1,311 @@
+This guide is for publishers who want to integrate with EUID and generate EUID tokens (advertising tokens) using only JavaScript client-side changes on their website with minimum effort.
 
-It includes the following sections:
+This guide does not apply to publishers who want to use a private operator, or who want to generate tokens server-side. Those publishers should follow the [Server-Side Integration Guide for JavaScript](integration-javascript-server-side.md).
 
-- [Introduction](#introduction)
-- [Integration Steps ](#integration-steps)
-  - [Establish Identity: User Login](#establish-identity-user-login)
-  - [Bid Using EUID Tokens](#bid-using-euid-tokens)
-  - [Refresh Tokens](#refresh-tokens)
-  - [Clear Identity: User Logout](#clear-identity-user-logout)
-- [FAQs](#faqs)
+EUID provides an SDK for JavaScript (see [SDK for JavaScript Reference Guide](../sdks/client-side-identity.md)) with the following features:
 
-## Introduction
+- EUID token generation
+- Automatic refreshing of EUID tokens
+- Automatic storage of EUID tokens in the browser
 
-This guide outlines the [basic steps](#integration-steps) that you need to consider for your integration. For example, you need to decide how to implement user login and logout, how to manage EUID identity information and use it for targeted advertising, how to refresh tokens, deal with missing identities, and handle user opt-outs. See also [FAQs](#faqs).
+You'll need to complete the following steps:
 
-To facilitate the process of establishing client identity using EUID and retrieving advertising tokens, the web integration steps provided in this guide rely on the [SDK for JavaScript](../sdks/client-side-identity.md).
+1. [Complete EUID account setup](#complete-euid-account-setup)
+2. [Add SDK for JavaScript to your site](#add-euid-sdk-for-javascript-to-your-site)
+3. [Configure the SDK for JavaScript](#configure-the-euid-sdk-for-javascript)
+4. [Check that the token was successfully generated](#check-that-the-token-was-successfully-generated)
 
->IMPORTANT: The SDK currently stores tokens in first-party cookies. Since implementation details like this may change in the future, to avoid potential issues, be sure to rely on the [SDK for JavaScript](../sdks/client-side-identity.md#api-reference) for your identity management.
+## SDK for JavaScript Version
 
-For integration scenarios for publishers that do not use the [SDK for JavaScript](../sdks/client-side-identity.md), see [Publisher Integration Guide, Server-Only (Without SDK)](custom-publisher-integration.md). 
+Support for client-side token generation is available in version 3.2 and above of the SDK. 
 
-## Integration Steps 
+The URL for the SDK is:
 
-The following diagram outlines the steps required for a user to establish a EUID token with a publisher and how the EUID token integrates with the RTB bid stream.
+- [https://cdn.prod.uidapi.com/uid2-sdk-3.2.0.js](https://cdn.prod.uidapi.com/uid2-sdk-3.2.0.js)
```

Comment:
This URL should be: https://cdn.prod.euid.eu/euid-sdk-3.3.0.js

Reply:
Done